### PR TITLE
recovery service soft delete

### PR DESCRIPTION
**Pull request by rvadeghar**

### CODE REVIEW

File: Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md [hunk 1]
```diff
@@ -561,4 +563,52 @@ Immutability must be enabled and locked on Recovery Services Vault
 
 ### Minimum permissions required to run the script
 Contributor or higher privileged role on the Redis Enterprise
+___
+
+## 35. Azure_BackupVault_DP_Enable_Immutability
+
+### Display Name
```
Comment (Aboli-msft) [inline Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md:570]: There should be single entry in this file for this control, recheck please

File: Scripts/RemediationScripts/Readme.md [hunk 2]
```diff
@@ -1422,6 +1423,81 @@ Yes
...
+### Supports rollback?
+No
+
+
+___
+
+
+## 69. Azure_RecoveryServicesVault_DP_Enable_Soft_Delete
+
+### Display Name
```
Comment (Aboli-msft) [inline Scripts/RemediationScripts/Readme.md:1490]: same as above

File: Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md (hunk 1 above)
Comment (rvadeghar) [reply Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md]: Removed duplication

File: Scripts/RemediationScripts/Readme.md (hunk 2 above)
Comment (rvadeghar) [reply Scripts/RemediationScripts/Readme.md:1431]: Removed duplication

**Review by Aboli-msft — APPROVED** [review]

**Review by MJaithra — APPROVED** [review]

**Pull request merged**